documentation/proposals/Proposal - Multi-Backend Input.md

```diff
@@ -179,13 +242,26 @@ public partial class Joysticks : IReadOnlyList<IJoystick>
 }
 ```
 
+All events will be raised when their matching actor methods are called, with the exception of `Click` and `DoubleClick` which are implemented on top of `ButtonDown` and `ButtonUp` respectively (as in 2.X).
+
+`DoubleClick` will be raised if `Mice.ButtonDown` is raised two consecutive times within `MouseClickConfiguration.DoubleClickTime` milliseconds, and the `MouseState.Position`'s `X` or `Y` did not change more than `MouseClickConfiguration.DoubleClickRange` between the two events. If these conditions are not met, `Click` is raised instead.
```

Comment:
Does `Click` have a time or range limit too, or will a hold-move-release trigger it?

Reply:
Hold-move-release won't register as a click the movement delta is within `MouseClickConfiguration.DoubleClickRange`. That being said, I was intending to copy and paste the 2.X implementation given how much work @Pyrdacor put into it so do have a play around with the click events in 2.X if you're curious.

Reply:
Maybe we should clarify this in the proposal? We could just say we'll keep their behavior as it is in 2.X.

Playing around with Windows 10's UI it doesn't seem like clicking has a time limit so scrap that

Reply:
Yep, I'll clarify in half an hour or so - just got back from the office and want to chill with a tea for a bit. Unless you're not around this evening? Then I'll do it now!

Reply:
A click is basically a mouse down which is not part of a double-click. So no matter for what reason a double-click "failed", the click event is raised. In general you will subscribe to a click event only if you want to distinguish it from a double-click. Otherwise just use mouse down. So these events are only useful if you implement click and double-click on the same target.

Reply:
Clarified.